modules/remotebackend/httpconnector.cc

```diff
@@ -183,9 +181,9 @@ void HTTPConnector::restful_requestbuilder(const std::string& method, const Json
   else if (method == "replaceRRSet") {
     std::stringstream ss2;
     for (size_t index = 0; index < parameters["rrset"].array_items().size(); index++) {
-      ss2 << buildMemberListArgs("rrset[" + std::to_string(index) + "]", parameters["rrset"][index]);
+      ss2 << buildMemberListArgs("rrset[" + std::to_string(index) + "]", parameters["rrset"][index]) << "&";
     }
-    req.body = ss2.str();
+    req.body = ss2.str().substr(0, ss2.str().size() - 1);
```

Comment:
```suggestion
    req.body = ss2.str().substr(0, ss2.str().size() - 1);  // remove trailing &
```

Reply:
Done. Thanks.

Reply:
great, thank you! Approved now, will merge soon once CI is green